### DISCUSSION TITLE
Re-synch MAC failed

### DISCUSSION THREAD

**Discussion by MerouaM** [herlesupreeth/docker_open5gs#338] [Q&A] [answered]
Hello all.

I am getting a "Re-synch MAC failed" error, when my UE tries to connect to the network.

I am using a docker deployment, along with UERANSIM for the RAN and UE part.

Please find attached the NF logs and the .env file.

[amf_log.txt](https://github.com/user-attachments/files/16176440/amf_log.txt)
[ausf_log.txt](https://github.com/user-attachments/files/16176430/ausf_log.txt)
[env.txt](https://github.com/user-attachments/files/16176431/env.txt)
[nrf_log.txt](https://github.com/user-attachments/files/16176432/nrf_log.txt)
[nr-gnb_log.txt](https://github.com/user-attachments/files/16176433/nr-gnb_log.txt)
[nr-ue_log.txt](https://github.com/user-attachments/files/16176434/nr-ue_log.txt)
[smf_log.txt](https://github.com/user-attachments/files/16176435/smf_log.txt)
[udm_log.txt](https://github.com/user-attachments/files/16176436/udm_log.txt)

Any clue on how to solve this ?

Thank you in advance.

Best regards

**Comment by herlesupreeth** [comment]
> I am getting a "Re-synch MAC failed" error, when my UE tries to connect to the network.

I would verify/check whether you have entered Ki and OP (**not OPc**) correctly in open5gs webui

**Reply by MerouaM** [reply, marked as answer]
Yes, you are right ! I was using OPc instead of OP when entering the SIM information in the webui. Now it works !

Thanks 😊
